### PR TITLE
Send all IP addresses of each interface (linux only)

### PR DESCRIPTION
**Pull request by mechairoi**

### CODE REVIEW

File: spec/linux/interface.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func (g *InterfaceGenerator) Generate() (interface{}, error) {
 		if iface["encap"] == nil || iface["encap"] == "Loopback" {
 			continue
 		}
-		if iface["ipAddress"] == nil && iface["ipv6Address"] == nil {
+		if len(iface["ipv4Addresses"].([]string)) == 0 && len(iface["ipv6Addresses"].([]string)) == 0 {
```
Comment (Songmu) [inline spec/linux/interface.go]: Please check the field is string slice or not.

**Comment by Songmu** [comment]
Almost okay. Please fix one point.

**Comment by mechairoi** [comment]
@Songmu  Thank you for the review. I also fixed the case `generateByIfconfigCommand`. Please review 671e710 :bow:

**Comment by Songmu** [comment]
LGTM!

**Comment by mechairoi** [comment]
Thanks :dash:

**Pull request merged**